### PR TITLE
Scroll on mobile search results

### PR DESCRIPTION
**Pull request by sczerwinski-wikia**
+ remove isPresent method from WikiBasePageObject

(mercury_search_loadMoreResultsOnSearchResultsPageNotVisible was failing because of the fact that search on mobile search results was disappearing once a scroll started, that resuted in scrolling more than it should by seach height)

http://jenkins.wikia-prod:8080/job/seleniumTestBuild_new/102/artifact/logs/log.html

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java [hunk 1]
```diff
@@ -131,10 +132,11 @@ public void scrollToElement(WebElement element) {
   private int getOffset() {
     WikiBasePageObject wikiPage = new WikiBasePageObject();
     int offset = wikiPage.getNavigationBarOffsetFromTop();
-    if (wikiPage.isBannerNotificationContainerPresent()) {
-      int notificationsHeight = wikiPage.getBannerNotificationsHeight();
-      offset += notificationsHeight;
-    }
+    if (wikiPage.isBannerNotificationContainerPresent()) offset += wikiPage.getBannerNotificationsHeight();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java]: MAJOR Missing curly brace. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00121)
MAJOR At most one statement is allowed per line, but 2 statements were found on this line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00122)

File: src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java [hunk 2]
```diff
@@ -131,10 +132,11 @@ public void scrollToElement(WebElement element) {
...
+
+    Search searchComponent = new SearchResultsPage().getSearch();
+    if (searchComponent.isPresent()) offset += searchComponent.getHeight();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java]: MAJOR Missing curly brace. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00121)
MAJOR At most one statement is allowed per line, but 2 statements were found on this line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00122)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues
* ![MAJOR][MAJOR] 4 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java [hunk 3]
```diff
@@ -131,10 +132,13 @@ public void scrollToElement(WebElement element) {
   private int getOffset() {
     WikiBasePageObject wikiPage = new WikiBasePageObject();
     int offset = wikiPage.getNavigationBarOffsetFromTop();
-    if (wikiPage.isBannerNotificationContainerPresent()) {
-      int notificationsHeight = wikiPage.getBannerNotificationsHeight();
-      offset += notificationsHeight;
-    }
+    if (wikiPage.isBannerNotificationContainerPresent())
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java]: MAJOR Missing curly brace. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00121)

File: src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java [hunk 4]
```diff
@@ -131,10 +132,13 @@ public void scrollToElement(WebElement element) {
...
+      offset += wikiPage.getBannerNotificationsHeight();
+
+    Search searchComponent = new SearchResultsPage().getSearch();
+    if (searchComponent.isPresent())
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java]: MAJOR Missing curly brace. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00121)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java (hunk 3 above)
Comment (nikodamn) [reply src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java]: +1

File: src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java (hunk 4 above)
Comment (nikodamn) [reply src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java]: +1

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java [hunk 5]
```diff
@@ -152,20 +152,7 @@ public String getUrl() {
...
-    return isElementPresent(BANNER_NOTIFICATION_CONTAINER);
-  }
-
-  /**
-   * Checks if present NOW
-   *
-   * TODO: Move this logic to better place (WebElement?)
-   */
-  public boolean isElementPresent(By selector){
-    changeImplicitWait(250, TimeUnit.MILLISECONDS);
```
Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java]: 😮

**Pull request merged**